en/chromebook_setup/instructions.md

```diff
@@ -53,7 +53,7 @@ Now you should see an interface with a sidebar, a big main window with some
 text, and a small window at the bottom that looks something like this:
 
 {% filename %}bash{% endfilename %}
-```
+```shell
```

Comment:
This is more like `console`, but that doesn't recognize a full prompt (like a full `PS1` variable: https://docs.pulpproject.org/pulpcore/configuration/settings.html#pulp-settings).